core/src/main/resources/reference.conf

```diff
@@ -535,6 +535,41 @@ datastax-java-driver {
     #   change.
     # Overridable in a profile: no
     warn-on-init-error = true
+
+
+    advanced-shard-awareness {
+      # Whether to use advanced shard awareness when trying to open new connections.
+      #
+      # Having this enabled makes sense only for ScyllaDB clusters.
+      # Results in smaller connection storms in multi-client settings.
```

Comment:
Does it ? I don't see any difference in this regard between regular shard awarness and advanced, if anything it will reduce number of reconnections needed to fullfill connection pool.

Reply:
If it decreases the number of reconnections needed then it lessens the load, no? Of course all driver applications will need to use it for best results.
Maybe I'm using a wrong term here?

Reply:
Can you please refrase it so that it does not confuse.

Reply:
`# Results in smaller connection storms in multi-client settings.` -> `Enabling it reduced amount of reconnections driver needs to do to get connection pool filled.`

Reply:
changed to `Reduces number of reconnections driver needs to fully initialize connection pool.`